app/visits/edit/template.hbs

```diff
@@ -226,6 +227,7 @@
                 </tr>
                 {{#each model.vitals as |vital|}}
                   <tr>
+                    <td>{{vital.modifiedBy}}</td>
```

Comment:
@jkleinsc, Actually, it is the user's display name.

The `modifiedBy` attribute is set here: https://github.com/HospitalRun/hospitalrun-frontend/blob/master/app/models/abstract.js#L86